dvc_s3/tests/fixtures.py

```diff
@@ -1,80 +1,20 @@
-import os
-
 import pytest
 
-from .cloud import S3, TEST_AWS_ENDPOINT_URL, TEST_AWS_REPO_BUCKET
```

Comment:
Could you check if these constants are used anywhere? Just in case they are trash and can be removed.

Reply:
So then TEST_AWS_REPO_BUCKET is used in real_s3, but TEST_AWS_ENDPOINT_URL is not used and can be removed, right?

Reply:
Yep, removed in the latest push :pray: